Classes/Configuration/Settings.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace JWeiland\Jwtools2\Configuration;
+
+/*
+ * This file is part of the solr_tool project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+/**
+ * Interface Settings
+ */
+interface Settings
```

Comment:
"Settings" kann man schnell mit $this->settings und {settings} verwechseln. Ich würde hier ein anderes Wording verwenden, wie z.B. "environment". Was meinst Du?